docs/root/version_history/current.rst

```diff
@@ -1,4 +1,4 @@
-1.22.3 (July 27, 2022)
+1.22.4 (Pending)
```

Comment:
due to ongoing release issues - 1.22.5 is about to be published - if we are quick we can include this

@jwendell could you rebase/merge #22556 so it doesnt conflict and we can merge

Reply:
on second thoughts - lets land this first and i can rebase